render/render.js

```diff
@@ -18,6 +18,15 @@ module.exports = function($window) {
 		return vnode.attrs && vnode.attrs.xmlns || nameSpace[vnode.tag]
 	}
 
+	// IE9 - IE11 (at least) throw an UnspecifiedError when accessing document.activeElement when
```

Comment:
Sorry, I don't understand what you're getting at -- if it's unable to retrieve the active element due to the access raising an exception it'll return `document.body`. Do you have a more sensible default in mind? I'm honestly out of my depth when it comes to the whole DOM and event model, so I'm happy to change it to whatever you think is reasonable.

Reply:
`document.body` is the `<body>` element's backing DOM node. If you think about that and where this function is called, that's why I'm requesting that you actually test it (and as mentioned in Gitter, your app with it).

If you don't yet know, you can generate a bundle via `npm run build-browser` and use the updated `mithril.js` file (don't commit it) in your app instead of the existing `mithril` bundle.

Reply:
Oh I'm sorry, I did actually test it against my app -- both in IE and Linux/Chrome. Thought that would go without saying :)

Reply:
Okay. Try testing it on a couple `option` nodes within a `select`, adding an `onchange` to the `select`, selecting the first option via the `selected` attribute, and then clicking on the other. If that works (I doubt it would), change `document.body` to `null` instead, so you aren't having to lookup a property.

Reply:
Yep, that works -- our app's interface relies pretty heavily on `onchange` and the various UI elements still update and do their thing.

I added some logging to the `activeElement` function and it looks like only the first 1-2 calls are triggering the exception, which is probably why nothing seems to be affected. I went ahead and changed the error case to return `null` as well per your suggestion.